editor/filesystem_dock.cpp

```diff
@@ -3289,25 +3294,31 @@ void FileSystemDock::_file_and_folders_fill_popup(PopupMenu *p_popup, const Vect
 		}
 	}
 
+	// Add the options that are only available when a single item is selected
```

Comment:
```suggestion
	// Add the options that are only available when a single item is selected.
```
Same in other comments.